Cargo.toml

```diff
@@ -58,7 +58,7 @@ tokio-util = { version = "0.7", features = ["rt"] }
 hyper-transport = ["dep:flume", "dep:hyper", "dep:postcard", "dep:bytes", "dep:tokio-serde", "tokio-util/codec"]
 quinn-transport = ["dep:flume", "dep:quinn", "dep:postcard", "dep:tokio-serde", "tokio-util/codec"]
 flume-transport = ["dep:flume"]
-iroh-net-transport = ["dep:iroh-net", "dep:flume", "dep:quinn", "dep:postcard", "dep:tokio-serde", "tokio-util/codec"]
+iroh-transport = ["dep:iroh-net", "dep:flume", "dep:quinn", "dep:postcard", "dep:tokio-serde", "tokio-util/codec"]
```

Comment:
`dep:iroh-net` reads odd, but I guess that needs to wait for the next release?

Reply:
Yeah, I do the rename now, and then we will have to fix this when the new crates get published.